lib/config.go

```diff
@@ -16,13 +16,15 @@ type ConfigMethods struct {
 	inst *Instance
 }
 
-// NewConfigMethods creates a configuration handle from an instance
-func NewConfigMethods(inst *Instance) *ConfigMethods {
-	return &ConfigMethods{inst: inst}
+// Name returns the name of this method group
+func (m *ConfigMethods) Name() string {
+	return "config"
 }
 
-// CoreRequestsName specifies this is a configuration handle
-func (m ConfigMethods) CoreRequestsName() string { return "config" }
+// ConfigMethods returns the `ConfigMethods` that the instance has registered
+func (inst *Instance) ConfigMethods() *ConfigMethods {
+	return &ConfigMethods{inst: inst}
+}
```

Comment:
`Instance` already has a method called `Config`. So this was given the name `ConfigMethods`.

Our other `MethodSet`s have the names `Access`, `Dataset`, and `Filesys`. So `ConfigMethods` is way less pretty. Any alternative names are welcome :)

Reply:
I was afraid we'd eventually run into this situation. As more and more functionality is moved from `Instance` into `scope`, it should be unnecessary to have so many accessors and methods publicly accessible on `Instance`; implementations should be getting what they need from `scope` instead. Over time the only public methods on `Instance` should be these method set accessors and whatever is needed to those using qri as a library. It might be the case that the current `Config()` method becomes unneeded by that point (it seems to only be used by `lib/remote.go`, `lib/resolve.go` and `api/` right now) but we shouldn't require fixing that before getting this PR merged.

I'm fine with using the slightly less pretty `ConfigMethods` for now. Could you just add a TODO to reevaluate this name as the dispatch refactor nears completion.